chainer/distributions/normal.py

```diff
@@ -0,0 +1,239 @@
+import chainer
+from chainer.backends import cuda
+from chainer import Distribution
+from chainer.functions.array import broadcast
+from chainer.functions.array import expand_dims
+from chainer.functions.array import repeat
+from chainer.functions.math import erf
+from chainer.functions.math import erfinv
+from chainer.functions.math import exponential
+import numpy
+
+
+class Normal(Distribution):
+
+    """Normal Distribution.
+
+    Args:
+        loc(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Parameter of distribution representing the \
+        location :math:`\\mu`.
+        scale(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Parameter of distribution representing the \
+        scale :math:`\\log(\\sigma^2)`.
+
+    """
+
+    def __init__(self, loc, scale):
+        super(Normal, self).__init__()
+        if isinstance(loc, chainer.Variable):
+            self.loc = loc
+        else:
+            self.loc = chainer.Variable(loc)
+        if isinstance(scale, chainer.Variable):
+            self.scale = scale
+        else:
+            self.scale = chainer.Variable(scale)
+
+    def __copy__(self):
+        return self._copy_to(Normal(self.loc, self.scale))
+
+    @property
+    def batch_shape(self):
+        return self.loc.shape
+
+    def cdf(self, x):
+        """Returns Cumulative Distribution Function for a input variable.
```

Comment:
No need to repeat the documentation if there is nothing special for this distribution (ditto for other methods and properties).